inspire_matcher/api.py

```diff
@@ -58,11 +59,17 @@ def match(record, config=None):
         config = current_app.config['MATCHER_DEFAULT_CONFIGURATION']
 
     try:
-        algorithm, doc_type, index = config['algorithm'], config['doc_type'], config['index']
+        algorithm = config['algorithm']
+        if ES_VERSION[0] == 7:
```

Comment:
shouldn't be ``>=``

Reply:
Yup it might be >= (let's hope they won't change their mind on version 8 :P )